apps/evm/src/pages/Market/Page/AssetWarning/index.tsx

```diff
@@ -63,20 +65,24 @@ export const AssetWarning: React.FC<AssetWarningProps> = ({
       <Modal
         isOpen={showAssets}
         handleClose={handleHideAssets}
-        noHorizontalPadding
+        noHorizontalPadding={isSmOrUp}
         title={t('assetWarning.modalTitle', {
           poolName: pool.name,
         })}
       >
         <MarketTable
           data-testid={TEST_IDS.marketTable}
           rowOnClick={handleHideAssets}
+          controls={false}
+          selectVariant={isSmOrUp ? 'tertiary' : 'quaternary'}
```

Comment:
It seems the select is only shown in the XS resolution, so I don't think you need to the ternary operator.
```suggestion
          selectVariant="quaternary"
```